js/components/frame.js

```diff
@@ -121,9 +121,11 @@ class Frame extends ImmutableComponent {
 
     let partition
     if (this.props.frame.get('isPrivate')) {
-      partition = 'private-1'
+      partition = 'default'
     } else if (this.props.frame.get('partitionNumber')) {
       partition = `persist:partition-${this.props.frame.get('partitionNumber')}`
+    } else {
+      partition = 'persist:default'
```

Comment:
that will definitely be lost as discussed in slack. Everyone seemed ok with it, but if not I can go another route. There is also currently an issue with ad insertion that this fixes for some reason.